tools/bazel.rc

```diff
@@ -38,3 +38,6 @@ build:clang-msan --copt -fsanitize=memory
 build:clang-msan --linkopt -fsanitize=memory
 build:clang-msan --define tcmalloc=disabled
 build:clang-msan --copt -fsanitize-memory-track-origins=2
+
+# Test options
+test --test_env=HEAPCHECK --test_env=PPROF_PATH
```

Comment:
I don't have a strong opinion on which one is chosen as long as we doc it. Will let both of you choose.